source/macos/AdaptiveCards/AdaptiveCards/Views/ACRTextField.swift

```diff
@@ -0,0 +1,104 @@
+import AdaptiveCards_bridge
+import AppKit
+
+class ACRTextField: NSTextField {
+    var isDarkMode: Bool = false
+    
+    init(frame frameRect: NSRect, config: RenderConfig) {
+        super.init(frame: frameRect)
+        initialise(config: config)
+        setupConstraints(config: config)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func initialise(config: RenderConfig) {
+        self.isDarkMode = config.isDarkMode
+        let myCell = VerticallyCenteredTextFieldCell()
+        let inputConfig = config.inputFieldConfig
+        myCell.setupSpacing(rightPadding: inputConfig.rightPadding, leftPadding: inputConfig.leftPadding, focusRingCornerRadius: inputConfig.focusRingCornerRadius, borderWidth: inputConfig.borderWidth)
+        self.cell = myCell
+        self.font = .systemFont(ofSize: inputConfig.fontSize)
```

Comment:
```suggestion
        cell = myCell
        font = .systemFont(ofSize: inputConfig.fontSize)
```
Nit: Use `self` if and only if needed.